datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -606,7 +609,7 @@ private void handleAssignmentChange(boolean isDatastreamUpdate) throws TimeoutEx
 
     List<String> deactivated = new ArrayList<>(oldConnectorList);
     deactivated.removeAll(newConnectorList);
-    List<Future<Boolean>> assignmentChangeFutures = deactivated.stream()
+    List<Future<Boolean>> assignmentChangeFutures = assignmentChangeFutures = deactivated.stream()
```

Comment:
I will fix this in a following update to this patch along with additional review comments.